### PR TITLE
Updated defaultTopLevelDomains to include relevant .org subdomains

### PR DESCRIPTION
**Pull request by williamdlees**
Created in response to issue #117: this follows the approach used for .co.uk, .co.nz etc and ensures that the default set of domains will not create spurious requests for commonly used .org subdomains.

### CODE REVIEW

File: src/mailcheck.js [hunk 1]
```diff
@@ -26,10 +26,12 @@ var Mailcheck = {
...
-  defaultTopLevelDomains: ["com", "com.au", "com.tw", "ca", "co.nz", "co.uk", "de",
-    "fr", "it", "ru", "net", "org", "edu", "gov", "jp", "nl", "kr", "se", "eu",
-    "ie", "co.il", "us", "at", "be", "dk", "hk", "es", "gr", "ch", "no", "cz",
-    "in", "net", "net.au", "info", "biz", "mil", "co.jp", "sg", "hu"],
+  defaultTopLevelDomains: ["com", "com.au", "org.au", "com.tw", "org.tw", "ca", 
+    "co.nz", "org.nz", "co.uk", "org.uk", "de", "fr", "it", "ru", "org.ru", "net", 
+    "org", "edu", "gov", "jp", "nl", "kr", "se", "org.se", "eu", "ie", "co.il", 
+    ".org.il", "us", "at", "be", "dk", "hk", "org.hk", "es", "org.es", "gr", "org.gr",
+    "ch", "no", "cz", "in", "org.in", "net", "net.au", "info", "biz", "mil", "co.jp", 
+    "sg", "org.sg", "hu", "org.hu"],
```
Comment (weilu) [inline src/mailcheck.js]: I don't think this is the way to go as we are just expanding the size of the default domains with not-so-common domain names. Perhaps include a paragraph on how to customize the domain list to cater to the org.[country] cases in the readme?

**Comment by williamdlees** [comment]
I would suggest that it's important to provide a generally useful default, for those cases where implementers don't take the time to analyze their signup addresses and determine which domains are most common (I think there are probably quite a few people in this category, perhaps even a majority, but I can't think of a good way of establishing that).  At the moment it looks to me as though the default list has grown somewhat organically. For example, is there a good reason that it includes .co.il but doesn't include .co.jp?

Things could get quite a bit messier with so many new top level domains being introduced now. Perhaps it is time to organize the defaults a little differently. For example, the default top level domains could be split into country domains, and other frequently used top level domains. If that were done, it would be easier for implementers to review and extend where necessary. It would also enable us to introduce a flag or option to include .co and .org subdomains of the country tlds, which would really reflect the way that things work a bit better. I would be happy to reorganize things along those lines if you like.

In the absence of a reorganization like that, I'd go back to the point I made in my original issue: it's unlikely that someone who types .org.ru or .org.il really meant to type .org, so it doesn't make much sense to offer .org as a suggestion. Meaningless suggestions make sites look bad. The default should offer reasonable suggestions to people who aren't prepared to put the work in to customize the settings. As the internet gets larger and more global, it seems inevitable that the list of defaults would get a bit larger.

File: src/mailcheck.js (hunk 1 above)
Comment (derrickko) [inline src/mailcheck.js]: +1

**Comment by derrickko** [comment]
We should definitely provide defaults. That said, whether they are truly useful is an open question, due to the very different requirements per site. As you've also mentioned, the number of TLDs keep growing, and the definition of a "useful default" get fuzzier and eventually hard to maintain. 

We want to encourage developers to base their implementation on datasets that matter to them. That's where mailcheck truly shines. The current defaults are ambiguously specific, and we'll likely cut it down to a very short list, with the intention to encourage this. We still maintain a set of domains for convenience in the [wiki](https://github.com/mailcheck/mailcheck/wiki), and we should start maintaining a list of TLDs (for use as defaults) there as well.

**Pull request closed without merging**